pages/common/ocrmypdf.md

```diff
@@ -1,27 +1,27 @@
 # ocrmypdf
 
 > Generate a searchable PDF or PDF/A from a scanned PDF or an image of text.
-> More information: <https://ocrmypdf.readthedocs.io/en/latest/introduction.html>.
+> More information: <https://ocrmypdf.readthedocs.io/en/latest/cookbook.html>.
 
 - Create a new searchable PDF/A file from a scanned PDF or image file:
 
-`ocrmypdf {{input_file}} {{output.pdf}}`
+`ocrmypdf {{path/to/input_file}} {{path/to/output.pdf}}`
 
-- Overwrite a scanned PDF file with a searchable PDF file, if successful:
+- Replace a scanned PDF file with a searchable PDF file:
 
-`ocrmypdf {{input.pdf}}`
+`ocrmypdf {{path/to/input.pdf}}`
 
 - Skip pages of a mixed-format input PDF file that already contain text:
 
-`ocrmypdf --skip-text {{input.pdf}} {{output.pdf}}`
+`ocrmypdf --skip-text {{path/to/input.pdf}} {{path/to/output.pdf}}`
 
-- Automatically clean, de-skew and rotate pages of a poor scan:
+- Clean, de-skew, and rotate pages of a poor scan:
 
-`ocrmypdf --clean --deskew --rotate-pages {{input_file}} {{output.pdf}}`
+`ocrmypdf --clean --deskew --rotate-pages {{path/to/input_file}} {{path/to/output.pdf}}`
 
 - Set the metadata of the searchable PDF file:
 
-`ocrmypdf --title "Document Title" --author "Document Author" --subject "Subject Description" --keywords "Multiple Keywords" {{input_file}} {{output.pdf}}`
+`ocrmypdf --title "{{title}}" --author "{{author}}" --subject "{{subject}}" --keywords "{{keywords}}" {{path/to/input_file}} {{path/to/output.pdf}}`
```

Comment:
This will probably be one of my last question/fixes

What is the exact format for keywords here?

is it just spaces between words, is it commas?

When using examples that could have 1 or more argument, it is 1 arg, 2nd arg then ... to signify that it is reoccurring if required

Reply:
So if you want to change it 

{{keyword_1 keyword_2 ...}}

(change the syntax of it if its not correct

Reply:
"{{keywords}}" is not specified by the author of ocrmypdf, that I have found. Adobe isn't much better. One adobe link from stackoverflow says "Keywords must be separated by commas or semicolons." I have added semicolons to the example. "{{keyword1;keyword2...}}"

Reply:
is that the syntax though, would you enter --keywords "alpha;bravo;charlie"

Reply:
Also I am reallllly being extreme, but space between the last keyword example and the ...

Reply:
Yes, you are, but that's the function of review. The format of the keyword metadata is not well defined by either the author of ocrmypdf or Adobe. It appears to be the subject of some discussion online. I have provided my best estimate based on what I have found.